README.md

```diff
@@ -33,20 +33,21 @@ are typically self-explanatory.
 4. Create a folder named "programming" in your computer's "Documents" directory.
   
 5. Open a new window in VS Code. Click "Open Folder". Find your programming folder. Click it once then click "Open".
-        a. To open a terminal in VS Code click "Shift + Ctrl + ~"
-        b. If at any point in your terminal entries the beginning line of your terminal starts with ">>", ">>>", or is
-           seemingly unable to enter your commands, click "Ctrl + C". That will quit any process running in your terminal.Sidebar- three arrows probably means you start a Python shell in your terminal! Another way to quit this type of process would be to type "quit()" and then enter.
```

Comment:
I think the lines of the bullets points a,b,c should all align ? under 5 there shouldn't be any sentences from the bullet points